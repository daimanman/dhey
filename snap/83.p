{"C":1,"Duration":6980808,"EndTime":1537240577287929800,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537240577280949000,"Status":1,"TaskId":83,"Type":"FILE","Url":"http://192.168.1.192:9091/lhcenter/api/v1/oss","Z":""}