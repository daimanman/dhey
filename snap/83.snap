{"AvgTotal":0.0069809,"Fastest":0.0069809,"Slowest":0.0069809,"Average":0.0069809,"Rps":143.2480052715266,"AvgConn":0.0009966,"AvgDNS":0,"AvgReq":0.0009657,"AvgRes":0,"AvgDelay":0.0029925,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0.0009657,"ReqMax":0.0009657,"ResMin":0.0009657,"ResMax":0,"DelayMax":0.0029925,"DelayMin":0.0029925,"Lats":[0.0069809],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0.0009657],"ResLats":[0],"DelayLats":[0.0029925],"Total":6980900,"ErrorDist":{},"StatusCodeDist":{"404":1},"SizeTotal":19,"SizeReq":19,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0069809,"Count":1,"Frequentcy":1},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0},{"Mark":0.0069809,"Count":0,"Frequentcy":0}],"TaskId":83,"Duration":0.0069809}